aim/web/api/runs/note/pydnamic_models.py

```diff
@@ -0,0 +1,6 @@
+from pydantic import BaseModel
```

Comment:
please move the APIs to `runs/views.py`
the directory structure becomes too nested here.
same for the inout models